<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>result</class>
 <widget class="QDialog" name="result">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1426</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="draw">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>30</y>
     <width>441</width>
     <height>611</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="method">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>161</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>method</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="firstfit">
        <property name="text">
         <string>First Fit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="bestfit">
        <property name="text">
         <string>Best Fit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="WorestFit">
        <property name="text">
         <string>Worest Fit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Deallocation &amp; memory content</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item>
     <widget class="QPushButton" name="deallocate">
      <property name="text">
       <string>deallocate</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Add new Process</string>
   </property>
   <widget class="QPushButton" name="push">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add process</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>98</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>number of segments</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="number">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>331</width>
     <height>611</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Neue Haas Grotesk Text Pro Medi</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Memory Layout</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>0</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Neue Haas Grotesk Text Pro</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Segmants tabel</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>171</width>
     <height>611</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>650</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
